src/handlers/monthly-post.js

```diff
@@ -4,13 +4,69 @@
  * A simple example includes a HTTP get method to get one item by id from a DynamoDB table.
  */
 
+function getLocalizedMonth(date, language) {
+  return date.toLocaleString(language, { month: 'long' });
+}
+
+function getLocalizedDate(date, language) {
+  return `${date.getDate()} (${date.toLocaleString(language, { weekday: 'short' })})`;
+}
+
+function getLocalizedMonthNames(dates, language) {
+  return localizedMonth = getLocalizedMonth(dates[0], language);
+}
```

Comment:
I don't really understand the difference between this and `getLocalizedMonth()`.

Reply:
Functionally, it was returning the month name for an array of `Date` objects instead of one. But I realized I can do this by just passing `dates[0]` when mapping `monthNames` like so:

```
en: months.map((dates) => getLocalizedMonth(dates[0], 'en')),
```

So I'm getting rid of this